test/DateBoundariesOptions.tsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { getISOLocalDateTime } from '@wojtekmaj/date-utils';
 
 type DateBoundariesOptionsProps = {
-  maxDate: Date | undefined;
-  minDate: Date | undefined;
-  setMaxDate: (date: Date | undefined) => void;
-  setMinDate: (date: Date | undefined) => void;
+  maxDate?: Date;
+  minDate?: Date;
+  setMaxDate: (date?: Date) => void;
+  setMinDate: (date?: Date) => void;
 };
```

Comment:
I think it would be cleaner to use `Optional Property`.

Reply:
There's a significant difference between "this property must be passed, but may be undefined somewhere down the line" and "you can forget about passing that property and it will be alright". It's purposeful.

Reply:
@wojtekmaj Wow, that's right, I get it.